### PR TITLE
Added shubhashisroydipta.com to README.md

### PR DESCRIPTION
**Pull request by dipta007**

### CODE REVIEW

**Review by george-gca — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -169,6 +170,7 @@ Workshop on Diffusion Models (NeurIPS: <a href="https://diffusionworkshop.github
 
 ## Table Of Contents
 
+- [al-folio](#al-folio)
```
Comment (pourmand1376) [inline README.md:173]: Is this change necessary?

**Comment by dipta007** [comment]
@pourmand1376 @george-gca fixed.
You can review it now.

**Review by george-gca — APPROVED** [review]
